uni/ulsp/workspace.icn

```diff
@@ -229,19 +349,77 @@ class Workspace(
                 move(2)
                 tab(many(' \t'))
                 className := tab(upto("\("))
-                if member(linked_classes, className) | member(imported_classes, className) | member(internal_classes, className) then {
+                if member(linked_classes, className) | member(imported_classes, className) | member(internal_classes, className) | member(package_classes, className) then {
                    objects[objectName] := className
-                   }
+                   } 
                 }
              }
           }
-       write(ximage(objects))
     end
 
     method getObjects()
        return objects
     end
 
+# -------------------------------------------------------------------------------------------------
+# ---------------- Variables (get, set, update) -----------------------------------------------------
+# -------------------------------------------------------------------------------------------------
+
+    method setVariables()
+       identifiers := &letters ++ &digits ++ '_' ++ '-'
```

Comment:
```
static identifiers 
initial identifiers := &letters ++ &digits ++ '_' ++ '-'
```
if we use this variable in several places, it might make sense to make it a class variable or package global and use it everywhere.